src/components/Countries.js

```diff
@@ -0,0 +1,73 @@
+import { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowCircleRight } from '@fortawesome/free-solid-svg-icons';
+import data from './data';
+import '../assets/country.css';
+
+function Countries() {
+  const [info, setInfo] = useState([]);
+  const [searchInput, setSearchInput] = useState('');
+  const [filteredInfo, setFilteredInfo] = useState([]);
+
+  useEffect(() => {
+    setInfo(data);
+    setFilteredInfo(data);
+  }, []);
+
+  const handleSearchInputChange = (event) => {
+    const inputValue = event.target.value;
+    setSearchInput(inputValue);
+
+    const filteredCountries = info.filter((country) => country.country.toLowerCase()
+      .includes(inputValue.toLowerCase()));
+    setFilteredInfo(filteredCountries);
+  };
+  /* eslint-disable */ 
+  return (
+    <div className="container" data-testid="countries-component">
+      <div className="search-area">
+        <input
+          type="text"
+          className="search-country"
+          placeholder="Search country"
+          value={searchInput}
+          onChange={handleSearchInputChange}
+        />
+      </div>
+      <div>
+        <div className="cuntries-container">
+          {filteredInfo.map((country) => {
+            return (
+              <div key={country.id} className="country-item">
+                <div key={country.alpha3}>
+                  <Link className="country-link" to={`/${country.country}`}>
+                    <FontAwesomeIcon className="icon-forward" icon={faArrowCircleRight} />
+                    <img
+                      loading="lazy"
+                      width={130}
+                      height={140}
+                      alt="card"
+                      className="img-fluid"
+                      src={country.map}
+                    />
+                    <h3 className="country-name">
+                      {country.country}
+                    </h3>
+                    <small className="text-neutral-100">
+                      ALPHA:
+                      {' '}
+                      {country.alpha3}
+                    </small>
+                  </Link>
+                </div>
+              </div>
```

Comment:
- [ ] It would be nice if you could match your design with the original design in the following points:
- On mobile screens, two countries cards are shown in a row and not one card as it's in your design. Could you work on your styles to achieve that with the help of a grid style? You could show two cards displayed and on desktop screens, you offer more as it is currently showing 👍🏽
- I highly recommend you change your app's font to use the Lato font from Google Fonts. Currently, you're using the default Times New Roman font. Please import and use the Lato font as required in the design guidelines to complete the look and feel of your font 👍🏽

**The Original Design Countries Cards**

<img width="417" alt="Screenshot 2023-08-25 at 8 13 16 AM" src="https://github.com/akbarsazish/Air-Pollution-API/assets/50941074/12558bad-c249-41a4-9e5c-73001c964fb5">

**Your Design**

<img width="470" alt="Screenshot 2023-08-25 at 8 12 23 AM" src="https://github.com/akbarsazish/Air-Pollution-API/assets/50941074/866d93c9-a094-45d5-8a8d-1059cae0ed93">

**Font Used**

<img width="1381" alt="Screenshot 2023-08-25 at 8 29 49 AM" src="https://github.com/akbarsazish/Air-Pollution-API/assets/50941074/d737f481-4c75-4068-92ff-978d84f9edeb">